specs/InfoBadge/InfoBadge-spec.md

```diff
@@ -0,0 +1,526 @@
+                                     InfoBadge="{x:Bind ItemInfoBadge}">
+            </muxc:NavigationViewItem>
+        </DataTemplate>
+    </muxc:NavigationView.MenuItemTemplate>
+</muxc:NavigationView>
+```
+ 
+C# Code-behind:
+```csharp
+public sealed partial class MainPage : Page
+{
+
+    public MainPage()
+    {
+        this.InitializeComponent();
+
+        
+        // Initialize icons for NavigationView items
+        SymbolIcon HomeIcon = new SymbolIcon(Symbol.Home);
+        SymbolIcon AcctIcon = new SymbolIcon(Symbol.Contact);
+        SymbolIcon MailIcon = new SymbolIcon(Symbol.Mail);
+
+        // Initialize an InfoBadge for the Mail item that is not visible on page load
+        InfoBadge MailBadge = new InfoBadge();
+        MailBadge.Value = 0;
+        MailBadge.IsOpen = false;
+
+        //Populate NavigationView
+        List<CustomNavObject> navItems = new List<CustomNavObject>(){
+            new CustomNavObject("Home", "home", HomeIcon),
+            new CustomNavObject("Account", "acct", AccountIcon),
+            new CustomNavObject("Mail", "mail", MailIcon, MailBadge)
+        };
+    }
+
+    // This function is called when a new message is received.
+    private void OnMessageReceived(string msg)
+    {
+        //Once a new message is received, update the InfoBadge being shown on the Mail navigation item. 
+           MailBadge.Value +=1;
+           MailBadge.IsOpen = true;
+    }
+}
+
+public class CustomNavObject
+{
+    public string Title { get; set; }
+    public string Tag { get; set; }
+    public IconSource ItemIcon { get; set; }
+    public InfoBadge ItemInfoBadge { get; set; }
+
+    public CustomNavObject(string title, string tag, IconSource icon)
+    {
+        Title = title;
+        Tag = tag;
+        ItemIcon =  icon;
+    }
+
+    public CustomNavObject(string title, string tag, IconSource icon, InfoBadge badge)
+    {
+        Title = title;
+        Tag = tag;
+        ItemIcon =  icon;
+        ItemInfoBadge = badge;
+    }
+
+}
+```
+
+# API Pages
+
+
+## InfoBadge class
+
+Represents a control that displays brief status as an icon, number, or dot. 
+
+`public class InfoBadge : Control`  
+
+### Example
+Here's an example of a simple InfoBadge that's set to display the number of unread emails:
+
+```xml
+<InfoBadge x:Name="emailInfoBadge" Value="{x:Bind numUnreadMail}"/>
+```
+### Remarks
+
+If neither `InfoBadge.Value` nor `InfoBadge.IconSource` are set, the InfoBadge defaults to showing a dot. If both of these properties are set, the value appears in a numeric InfoBadge. 
+
+If `InfoBadge.IconSource` is set to null, the InfoBadge defaults to displaying a dot. If `InfoBadge.Value` is set to -1, InfoBadge also defaults to displaying a dot. In order to hide the InfoBadge, the `InfoBadge.IsOpen` property needs to be set to False. 
+
+
+## InfoBadge.Value property
+
+Gets or sets the integer to be displayed in a numeric InfoBadge.
+
+` public int InfoBadge.Value { get; set; }`
```

Comment:
Would it be better to be a nullable int?

Reply:
I think it was brought up that there were issues with nullable types and binding, but this would be good to discuss during the API review.